README.md

```diff
@@ -112,6 +113,28 @@ function ObservePerson(props) {
 }
 ```
 
+### `useObserver<T>(fn: () => T, baseComponentName = "anonymous"): T`
+
+Low level implementation used internally by `observer`.
+It allows you to use an `observer` like behaviour, but still allowing you to optimize the component in any way you want (e.g. using `memo` with a custom
+`areEqual`, using `forwardRef`, etc.) and to declare exactly the part that is observed (the render phase). One good thing about this is that if any
```

Comment:
nit ... remove line break here and the line above, please :)

Reply:
done